src/evm/mod.rs

```diff
@@ -359,7 +365,7 @@ pub fn evm_test_arbsys_direct(log_to: Option<&Path>, debug: bool) -> Result<(),
     assert_eq!(my_addr.clone(), my_addr_decompressed);
     assert_eq!(offset, Uint256::from_usize(my_addr_compressed.len()));
 
-    assert_eq!(Uint256::from_u64(2), arb_address_table.size(&mut machine)?);
+    assert_eq!(Uint256::from_u64(3), arb_address_table.size(&mut machine)?);
```

Comment:
I don't see any clear reason why this has changed?

Reply:
There was a change in decompression.mini, line 155, which causes the decompressor to add addresses to the address table more aggressively.  In this test case, one more address gets added, compared to before.